packaging/bundle-validation/base/Dockerfile

```diff
@@ -51,9 +52,16 @@ RUN wget https://archive.apache.org/dist/flink/flink-$FLINK_VERSION/flink-$FLINK
     && rm $WORKDIR/flink-$FLINK_VERSION-bin-scala_2.12.tgz
 ENV FLINK_HOME=$WORKDIR/flink-$FLINK_VERSION
 
-RUN wget https://archive.apache.org/dist/spark/spark-$SPARK_VERSION/spark-$SPARK_VERSION-bin-hadoop$SPARK_HADOOP_VERSION.tgz -P "$WORKDIR" \
-    && tar -xf $WORKDIR/spark-$SPARK_VERSION-bin-hadoop$SPARK_HADOOP_VERSION.tgz -C $WORKDIR/ \
-    && rm $WORKDIR/spark-$SPARK_VERSION-bin-hadoop$SPARK_HADOOP_VERSION.tgz
+RUN if [ "$SCALA_VERSION" = "2.13" ]; then \
```

Comment:
Spark version is set to 3.1.3 so do we need this?

Reply:
That's the default value: `ARG SPARK_VERSION=3.1.3`.  When we build the docker image, we override the arguments as needed, e.g., see `packaging/bundle-validation/base/build_flink1180hive313spark350scala213.sh`:
```
docker build \
 --build-arg HIVE_VERSION=3.1.3 \
 --build-arg FLINK_VERSION=1.18.0 \
 --build-arg SPARK_VERSION=3.5.0 \
 --build-arg SPARK_HADOOP_VERSION=3 \
 --build-arg HADOOP_VERSION=3.3.5 \
 --build-arg SCALA_VERSION=2.13 \
 -t hudi-ci-bundle-validation-base:flink1180hive313spark350scala213 .
```